src/patternfly/demos/Toolbar/examples/toolbar-complex-example.hbs

```diff
@@ -0,0 +1,99 @@
+{{#> toolbar }}
+    {{#> toolbar-section pf-l-toolbar--modifier="pf-u-justify-content-space-between pf-u-mx-xl pf-u-my-md" aria-label="First section"}}
+        {{#> toolbar-group}}
+            {{#> toolbar-item pf-l-toolbar__item--modifier="pf-u-mr-2xl"}}
+                {{#> input-group }}
+                    {{#> dropdown id="dropdown-example-collapsed"}}
+                        All
+                        <i class="fas fa-caret-down pf-c-dropdown__toggle-icon"></i> 
+                    {{/dropdown}}
+                {{#> input-group-action input-group-action--modifier="pf-m-align-right" inputGroupTextType="label" aria-label="popover for input"}}
+                    <i class="fas fa-search aria-hidden="true"></i>
+                {{/input-group-action}}
+                {{#> form-control controlType="input" input="true" type="text" id="textInput10" name="textInput10" placeholder="Filter..." aria-label="input example with popover" }}
+                {{/form-control}}
+                {{/input-group}}
+            {{/toolbar-item}}
+            {{#> toolbar-item pf-l-toolbar__item--modifier="pf-u-mr-lg"}}
+                {{#> dropdown id="dropdown-example-collapsed" pf-c-dropdown--HasToggleIcon="true"}}
+                    All
+                {{/dropdown}}
+            {{/toolbar-item}}
+            {{#> toolbar-item pf-l-toolbar__item--modifier="pf-u-mr-lg"}}
+                <button class="pf-c-button pf-m-plain"><i class="fas fa-sort-alpha-down"></i></button>
```

Comment:
Any of the buttons icons should also include a `aria-label=""` which you can see in the example in the buttons section. Sorry I should have caught that for your last round of changes.

Reply:
fixed